conf/app.routes

```diff
@@ -2,7 +2,7 @@
 GET        /epaye-auth/agent/:agentCode/client/:empRef          @uk.gov.hmrc.agentaccesscontrol.controllers.AuthorisationController.isAuthorisedForPaye(agentCode: uk.gov.hmrc.domain.AgentCode, empRef: uk.gov.hmrc.domain.EmpRef)
 GET        /sa-auth/agent/:agentCode/client/:saUtr              @uk.gov.hmrc.agentaccesscontrol.controllers.AuthorisationController.isAuthorisedForSa(agentCode: uk.gov.hmrc.domain.AgentCode, saUtr: uk.gov.hmrc.domain.SaUtr)
 GET        /mtd-it-auth/agent/:agentCode/client/:mtdItId        @uk.gov.hmrc.agentaccesscontrol.controllers.AuthorisationController.isAuthorisedForMtdIt(agentCode: uk.gov.hmrc.domain.AgentCode, mtdItId: uk.gov.hmrc.agentmtdidentifiers.model.MtdItId)
-GET        /afi/agent/:agentCode/client/:nino                   @uk.gov.hmrc.agentaccesscontrol.controllers.AuthorisationController.isAuthorisedForAfi(agentCode: uk.gov.hmrc.domain.AgentCode, nino: uk.gov.hmrc.domain.Nino)
+GET        /afi-auth/agent/:agentCode/client/:nino              @uk.gov.hmrc.agentaccesscontrol.controllers.AuthorisationController.isAuthorisedForAfi(agentCode: uk.gov.hmrc.domain.AgentCode, nino: uk.gov.hmrc.domain.Nino)
```

Comment:
How do we plan to orchestrate the change with **auth**, I think we should keep old version for a while, make PR to **auth** and remove old endpoint only after theirs release to prod, wdyt?

Reply:
So we are in the progress of getting a PR up for Auth. To begin with, we won't be changing the Prod config. Merging the Dev config will allow us to test the changes up to staging. As for the Prod change, we ca raise a PR and talk to the auth team and co-ordinate with them as to when we need the change deployed to Prod.

Reply:
I was wrong, sorry, mislead this URL for the same in agent-fi-access-control